app/controllers/courses_controller.rb

```diff
@@ -67,8 +67,9 @@ def course
 
   def course_params
     params.require(:course).permit(
-    :title,
-    :description
-    )
+                                  :title,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.